man/en/sos-upload.1

```diff
@@ -0,0 +1,151 @@
+.TH UPLOAD 1 "December 2024"
+
+.SH NAME
+sos_upload \- Upload files like previously generated sos reports or logs to a policy specific location
+.SH SYNOPSIS
+.B sos upload FILE [options]
+          [--case-id id]\fR
+          [--upload-url url]\fR
+          [--upload-user user]\fR
+          [--upload-pass pass]\fR
+          [--upload-directory dir]\fR
+          [--upload-method]\fR
+          [--upload-no-ssl-verify]\fR
+          [--upload-protocol protocol]\fR
+          [--upload-target target]\fR
+          [--upload-s3-endpoint  endpoint]\fR
+          [--upload-s3-region region]\fR
+          [--upload-s3-bucket bucket]\fR
+          [--upload-s3-access-key key]\fR
+          [--upload-s3-secret-key key]\fR
+          [--upload-s3-object-prefix prefix]\fR
+
+.PP
+.SH DESCRIPTION
+upload is an sos subcommand to upload sos reports,
+logs, vmcores, or other files to a policy defined
+remote location, or a user defined one.
+.SH REQUIRED ARGUMENTS
+.B FILE
+.TP
+The path to the archive that is to be uploaded.
+.SH OPTIONS
+.TP
+.B \--case-id NUMBER
+Specify a case identifier to associate with the archive.
+Identifiers may include alphanumeric characters, commas and periods ('.').
```

Comment:
Is it worth saying here that this is RH only?

Reply:
With `sos report` it does on all sosreport tarballs, and adds the case-id to the tarball filename at creation. It's more for upload. On the Canonical side, the case-id is not used for upload at all

Reply:
I can specify that's for RH only, I think is a good clarification

Reply:
I've added a small clarification in the man page